### DISCUSSION TITLE
Dynamic values

### DISCUSSION THREAD

**Discussion by arthurgeek** [tailwindlabs/tailwindcss#7104] [Help] [answered]
The docs for Tailwind v2 JIT mode had this disclaimer:

`Arbitrary values cannot be computed from dynamic values`.

I can't find anywhere on the new docs anything about this, is this still the case for v3?

**Comment by adamwathan** [comment, marked as answer]
Hey, docs here: https://tailwindcss.com/docs/content-configuration#dynamic-class-names

**Reply by arthurgeek** [reply]
Thanks for the fast reply!
